vulnfeeds/cmd/mirror-debian-copyright/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:22.04
```

Comment:
Since the command only needs `wget` and `gsutil`, I was wondering if there's an existing prebuilt image that we might be able to use instead?

Reply:
Maybe https://hub.docker.com/r/google/cloud-sdk/ alpine?

Reply:
It appears that on both the metrics of size and number of detected vulnerabilities, `ubuntu:22.04` wins.

I didn't try alpine, but I tried `google/cloud-sdk:latest` and `google/cloud-sdk:slim` because I'm a Debian weenie.

Reply:
`gcr.io/google.com/cloudsdktool/google-cloud-cli:alpine` appears to win in terms of size and detected vulnerabilities. I hope the latter isn't just because the scanner can't scan Alpine-based images...

Reply:
FWIW: When pushing a container to the registry, I'm pretty sure each Docker 'layer' is hashed and the registry and only one copy of layers with the hash is stored.

Since apt produces logs, the layers would not be the same each run, and the whole image would be stored each time.

If we use a prebuilt base image, the layers would be the same as long as the underlying base image is the same, so it only needs to store it once (and, if the rest of the layers do not change, the image we build would be identical to the previous version, and would not use any more storage)

(I'm not 100% certain on this...)